scripts/short_sem_to_long_sem.py

```diff
@@ -7,9 +7,9 @@
 
 if term == "01":
     long_term = "spring"
-elif term == "05":
+elif term == "05" or term == "02":
```

Comment:
I think there should be some comments explaining that the `02` and `03` are oxy-specific.  They'll make it easier to remember what is needed by-school once we get a more generic school specialization system setup.  The comments should be added here, in `src/utilities.ts`, and `vue.config.js`.